src/app/shell/workshop-details/side-menu/side-menu.component.html

```diff
@@ -1,5 +1,5 @@
-<div fxLayout="column" fxLayout.lt-lg="row wrap" fxLayoutAlign="start center" fxLayoutAlign.lt-lg="space-between">
-  <app-actions [workshop]="workshop"></app-actions>
+<div *ngIf="isDisplayed" fxLayout="column" fxLayout.lt-lg="row wrap" fxLayoutAlign="start center" fxLayoutAlign.lt-lg="space-between">
```

Comment:
*ngIf="isDisplayed"  should be applied only for app-actions components. You should not hide all the components of the side menu for the provider. Otherwise for provider it is looked like this 
![image](https://user-images.githubusercontent.com/27493959/126153392-e8c292d3-f0ac-45b1-a1ec-910713bb570c.png).

Move this condition to app-actions